test/unit/vec/src/VecTest.cpp

```diff
@@ -262,6 +262,32 @@ TEST_CASE("basicVecTraits", "[vec]")
             STATIC_REQUIRE(referenceVec == vecLessEqual);
         }
 
+        // alpaka::Vec elementwise_min
+        {
+            constexpr auto vecMin = alpaka::elementwise_min(vec, vec3);
+            STATIC_REQUIRE(std::is_same_v<alpaka::Dim<std::decay_t<decltype(vecMin)>>, Dim>);
+            STATIC_REQUIRE(std::is_same_v<alpaka::Idx<std::decay_t<decltype(vecMin)>>, bool>);
+
+            constexpr alpaka::Vec<Dim, Idx> referenceVec(
+                static_cast<Idx>(0u),
+                static_cast<Idx>(8u),
+                static_cast<Idx>(3u));
+            STATIC_REQUIRE(referenceVec == vecMin);
+        }
```

Comment:
Consider adding tests for calling `elementwise_min` with a single argument.
Please also add a test calling it with more than 2 arguments. Thx!

Reply:
Done.